src/main/webapp/WEB-INF/ViewStatistics.jsp

```diff
@@ -0,0 +1,67 @@
+<%@ page import="java.sql.*,java.util.Random,java.io.*"%>
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+	pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="ISO-8859-1">
+<title>View Statistics</title>
+</head>
+<body>
+<%
+
+Connection conn;
+
+try {
+	// obtain a connection to the DB, use DB driver, URL, credentials
+	String connURL = "jdbc:mysql://localhost:3306/PAGE_VISITS";
+	Class.forName("com.mysql.jdbc.Driver").newInstance();
+	conn = DriverManager.getConnection(connURL, "exercise1", "exEr(1se$");
```

Comment:
change the following to this:

	String connURL = "jdbc:mysql://172.18.0.1:3308/PAGE_VISITS";
	Class.forName("com.mysql.jdbc.Driver").newInstance();
	conn = DriverManager.getConnection(connURL, "root", "helloworld");

Reply:
Should be OK now.